plugins/rapid7_intsights/unit_test/util.py

```diff
@@ -30,6 +30,10 @@ def read_file_to_string(filename):
         with open(filename) as my_file:
             return my_file.read()
 
+    @staticmethod
+    def read_file_to_dict(filename):
+        return json.loads(Util.read_file_to_string(os.path.join(os.path.dirname(os.path.realpath(__file__)), filename)))
```

Comment:
Should we break this out into multiple lines or does it make sense as it is?

Reply:
I split it into new lines